models/wrf_hydro/model_mod.f90

```diff
@@ -250,13 +242,13 @@ subroutine static_init_model()
 
       call configure_hydro()
       call read_hydro_global_atts(domain_shapefiles(domainID))
-      call verify_variables(hydro_variables, domain_shapefiles(domainID), &
-                  n_hydro_fields, var_names, var_qtys, var_ranges, var_update)
+
+      call get_state_variables(hydro_variables, MAX_STATE_VARIABLES, use_clamping, state_vars_hydro)
```

Comment:
It is worth thinking about the name 'get_state_variables'.
Here it is called for three domains, so it is only 'getting' info for one domain, not the whole state.

I think also  it is good to think about what the routine is doing. It is not getting the data of any netcdf variables, it is just converting a table of characters to somehting that can be passed to add_domain.

Reply:
@hkershaw-brown do you have any suggestions for the name?